sync/class.jetpack-sync-module-updates.php

```diff
@@ -195,18 +199,43 @@ public function validate_update_change( $value, $expiration, $transient ) {
 		$checksums[ $transient ] = $new_checksum;
 
 		update_option( self::UPDATES_CHECKSUM_OPTION_NAME, $checksums );
-		/**
-		 * jetpack_{$transient}_change
-		 * jetpack_update_plugins_change
-		 * jetpack_update_themes_change
-		 * jetpack_update_core_change
-		 *
-		 * @since 5.1.0
-		 *
-		 * @param array containing info that tells us what needs updating
-		 *
-		 */
-		do_action( "jetpack_{$transient}_change", $value );
+		if ( 'update_core' === $transient ) {
+			/**
+			 * jetpack_update_core_change
+			 *
+			 * @since 5.1.0
+			 *
+			 * @param array containing info that tells us what needs updating
+			 *
+			 */
+			do_action( "jetpack_update_core_change", $value );
+			return;
+		}
+		if ( empty( $this->updates ) ) {
+			add_action( 'shutdown', array( $this, 'sync_last_event' ), 9 );
+		}
+		if ( ! isset( $this->updates[ $transient ] ) ) {
+			$this->updates[$transient] = array();
+		}
+		$this->updates[$transient][] = $value;
+	}
+
+	public function sync_last_event() {
+		foreach( $this->updates as $transient => $values ) {
+			$value = end( $values ); // only send over the last value
+			/**
+			 * jetpack_{$transient}_change
+			 * jetpack_update_plugins_change
+			 * jetpack_update_themes_change
+			 *
+			 * @since 5.1.0
+			 *
+			 * @param array containing info that tells us what needs updating
+			 *
+			 */
+			do_action( "jetpack_{$transient}_change", $value );
```

Comment:
should we send one that is more generic? `jetpack_updates_change`

Reply:
No I want to keep the actions the same as before so that we don't need changes on .com